build-system/tasks/extension-helpers.js

```diff
@@ -778,14 +778,8 @@ async function generateBentoEntryPointSource(name, toExport, options) {
 
   return dedent(`
     import {BaseElement} from '../base-element';
-    
+
     function defineElement() {
-      const css = __css__;
```

Comment:
Shouldn't `__css__` and all its references be deleted as well then?

Reply:
ah forgot to remove the rest of the references - updated